packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -48,6 +48,8 @@ import {
   scopePollingDetectionStrategy,
   isRedirectable,
   generalizedErrorMessage,
+  getAptosConfig,
+  isAptosNetwork,
```

Comment:
how come the linter doesn't organize these?

Reply:
It doesnt have this rule